Update README.md (#1)
withstudiocms/.github: profile/README.md

<h1 align="center"><strong>Welcome to @withstudiocms 👋</strong></h1>
<p align="center">
  <strong>withstudiocms</strong> is a community driven organization who's goal and overall mission is to bring

>     the best, community built tools that are free and open-source for all to use. Starting from just a small CMS

Reviewer: ```suggestion
    the best, community built tools that are free and open-source for all to use. Starting from our flagship project, StudioCMS,
```

>     project, now expanding and growing into multiple tools and utilies for the greator <a href="https://astro.build">Astro</a> eco-system!

Reviewer: ```suggestion
    now expanding and growing into multiple tools and utilities for the greater <a href="https://astro.build">Astro</a> eco-system!
```

</p>

<p align="center">

>   Have something to suggest? Looking for projects to get involved in? Checkout our <a href="https://github.com/withstudiocms/roadmap">roadmap repository</a> 

Reviewer: ```suggestion
  Have a suggestion? Or looking for projects to get involved in? Checkout our <a href="https://github.com/withstudiocms/roadmap">roadmap repository</a> 
```

  to get started in the discussions, or help work on an RFC.
</p>

<p align="center">
  <strong>Join the adventure</strong> and become a contributor by checking out our repositories
    or joining our <a href="https://chat.studiocms.dev">Discord</a>!
</p>

## Our Projects

### [`studiocms`](https://github.com/withstudiocms/studiocms)
[![NPM Version](https://img.shields.io/npm/v/studiocms?style=for-the-badge&logo=npm)](https://npm.im/studiocms)

> A dedicated CMS for Astro DB. Built from the ground up by the Astro community.

Reviewer: ```suggestion
A dedicated CMS for AstroDB. Built from the ground up by the Astro community.
```

### [`@studiocms/ui`](https://github.com/withstudiocms/ui)
[![NPM Version](https://img.shields.io/npm/v/%40studiocms%2Fui?style=for-the-badge&logo=npm)](https://npm.im/@studiocms/ui)

> The UI library for StudioCMS. Includes the layouts & components we use to build StudioCMS.

Reviewer: ```suggestion
The offcial UI library for StudioCMS. Includes the layouts & components we use to build StudioCMS. All with zero framework dependencies.
```

### [`@studiocms/markdown-remark`](https://github.com/withstudiocms/markdown-remark)
[![NPM Version](https://img.shields.io/npm/v/@studiocms/markdown-remark?style=for-the-badge&logo=npm)](https://npm.im/@studiocms/markdown-remark)

> Markdown parser and transformer built on top of remark. It provides seamless integration with Astro through an included astro integration, allowing you to easily parse and transform Markdown content (local or remote) within your Astro projects.

Reviewer: ```suggestion
A Markdown parser and transformer built on top of remark. It provides seamless integration with Astro through an Astro integration, allowing you to easily parse and transform Markdown content (local or remote) within your Astro projects.
```
